tests/docs/running-e2e-test.md

```diff
@@ -51,6 +51,12 @@ make docker-push
 make docker-deploy-k8s
 ```
 
+### Apply the test config
+
```

Comment:
Please add more comment to explain why we need.

Reply:
ok